apps/website/components/button/README.md

```diff
@@ -156,31 +156,31 @@ Icon buttons are useful where interface space may be limited. If an icon represe
 
 We recommend you **choose an icon that best describes the action** that the user will be doing. Users may avoid unknown or abstract icon buttons.
 
-#### Types & Sizes
+### Types & Sizes
 
 <div class="clr-row">
 
 <div class="clr-col-sm-12 clr-col-lg-6 doc-do">
 <div class="doc-example">
-<cds-button size="icon" style="margin-right: 12px"><cds-icon shape="check"></cds-icon></cds-button>
-<cds-button size="icon" action="outline" style="margin-right: 12px"><cds-icon shape="folder"></cds-icon></cds-button>
-<cds-button size="icon" action="flat"><cds-icon shape="cog"></cds-icon></cds-button>
+<button class="btn btn-primary btn-icon" style="margin-right: 12px"><clr-icon shape="check"></clr-icon></button>
```

Comment:
All these inline pixels values will break if the browser is zoomed in or out. That creates an a11y challenge.

Why didn't we use `0.6rem` instead of `12px`?

Reply:
Copied from existing site, will revise.